src/hotspot/share/gc/shenandoah/shenandoahPacer.hpp

```diff
@@ -27,6 +27,7 @@
 
 #include "gc/shenandoah/shenandoahNumberSeq.hpp"
 #include "gc/shenandoah/shenandoahPadding.hpp"
+#include "gc/shenandoah/shenandoahSharedVariables.hpp"
```

Comment:
Check this again.

Reply:
Needed for `ShenandoahSharedFlag` (at least). A previous include somewhere else satisfied all previous uses which unravelled when some of the files (notably shenandoah thread local data's .cpp file was moved to its own file).

Leaving as is.